### PR TITLE
feat: fixed, config file not found in relative path

### PR DESCRIPTION
**Pull request by photowey**
// config/boot.go
yamlFile, err := ioutil.ReadFile(configPath) // use abs
see [read-files](https://golangbot.com/read-files/)

### CODE REVIEW

File: config/boot_test.go [hunk 1]
```diff
@@ -53,17 +54,53 @@ func TestLoad(t *testing.T) {
...
+		wantErr             bool
+	}{
+		{
+			"test load from file",
+			filepath.Join(wd, "/test/ioc_golang.yaml"), // abs path
+			false,
+		},
+		{
+			"test load from file",
+			"/test/ioc_golang.yaml", // relative path
```
Comment (LaurenceLiZhixin) [inline config/boot_test.go:81]: Should this relative path be   './test/ioc-golang.yaml' ? 
@photowey

**Review by LaurenceLiZhixin — CHANGES_REQUESTED** [review]
Thanks for your pr, please sign the cla and check ut.

File: config/boot_test.go (hunk 1 above)
Comment (photowey) [reply config/boot_test.go]: This is a typo, when written as: "/test/ioc_golang.yaml", boot#IsNotAbsPath() returns true, so the execution result will be the same as the absolute path. At the same time, the unit test cases have been modified.

**Comment by LaurenceLiZhixin** [comment]
Thanks for your suggestions. I looked at your PR and I don't think it makes sense to pass relative paths through environment variables: environment variable can be read by all process and is of operating system  level. 
Relative path is of process level.
I think the relative path should be configured in the form of an API. For example, add config option to the config.load() function as param.

Here is my proposal:  https://github.com/alibaba/IOC-Golang/issues/11
@photowey

**Comment by photowey** [comment]
I think your suggestion is very reasonable, so the loading option is designed. see the latest commit, please.

**Comment by LaurenceLiZhixin** [comment]
Thanks for your design. I reviewed and found that you added load options, util functions, and multi file support. That's great !

My last commit on your branch fixed ci, added license headers and some logs.

I think there is one place needed to be modified. Can we combine the LoadOption function and Load function into one? If options is empty, load with default. What do you think? @photowey

**Comment by photowey** [comment]
The previous design did not support the way environment variables were originally adopted, this has been fixed. The test cases below config/boot_test.go can be run correctly.

**Review by LaurenceLiZhixin — APPROVED** [review]

**Pull request merged**